cmd/brew-command-not-found-init.rb

```diff
@@ -38,17 +43,9 @@ def help
 
 def init
   case shell
-  when :bash, :zsh
```

Comment:
Please don’t change this part; it’ll break the script for existing users.

Reply:
I was only able to use command-line because of an issue. Sorry for the late response! I have reversed those changes. Anything else I need to do?